sync-core/src/main/java/com/cloudant/sync/sqlite/SQLDatabaseQueue.java

```diff
@@ -42,6 +45,8 @@
      */
     public SQLDatabaseQueue(String filename) throws IOException {
         this.db = SQLDatabaseFactory.createSQLDatabase(filename);
+        //can directly add to queue no need to check
```

Comment:
But still doesn't explain why there's no need to check, and it is ambiguous what the `it` refers to in the following line, the `queue` or the directly added thing (presumably the runnable?).